Monika After Story/game/script-topics.rpy

```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
+    monika_random_topics.append('monika_colorblindness')
+	
+label monika_colorblindness:
+	m "Hey [player], I've been reading up a bit on color blindness."
+	m "I just can't imagine how it must be like!"
+	m "To never be able to see the roaring sun by its true colors or its gentle sunset!"
+	m "I really feel bad for those poor people."
+	m "Most of them are male too, since it's quite rare for a woman to have a color blind deficit of some sort."
+	m "Gosh, genetics are super weird, aren't they?"
+	m "Why don't we do a little experiment?"
+	menu:
+		m "What are the color of my eyes?"
+		"Green":
+			m "Ahaha, you're right!"
+			m "Well, at least that means you're not color blind."
+			m "Unless you lied to me, but why would you?"
+			m "Ah, in that case you, maybe just didn't want to worry me."
+			m "I'm always willing to deal with any of your worries [player]!"
+		"Gray":
+			m "Gosh, we might have an issue here."
+			m "If you see everything on a scale of greys, that could mean you're completely color blind!”
+			m "Unless you just have a super weird black and white monitor.
+			m "But take care of yourself."
+		"Other":
+			m "Ah, I can't tell from here."
+			m "I assume you either have a faulty monitor or you're color blind."
+			m "Yet there're so many different types of color blindness that I can't guess by just asking you."
+			m "You should seek professional advice in that case."
+			m "But if you aren't convinced yet, then try to look at an object with somebody else to find out if you see the same thing as different colors!"
+			m "You really should try to figure this out though, it might be good for you to know this information for the rest of your life!"
+			m "But rest assured, I'll love you no matter what issues you go through [player]!"
+	return
+    
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['waiting lines', 'waiting line', 'queues', 'queue']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_waitinglines') #id
+    monika_random_topics.append('monika_waitinglines')
+	
+label monika_waitinglines:
+	m "Hey [player], have you ever thought about how much time you've spent waiting in lines before?”
+	m "I mean, across your entire life of course!"
+	m "It's probably quite a long time, you know?"
+	m "I bet that time would double if we were to count more than just physical lines!"
+	m "What I mean by that is there's loads of different types of lines!"
+	m "You got those times where you get a number and sit down, waiting for your number to be called!"
+	m "Those might sound rather calm but honestly, there's an even better option!"
+	m "You could just get a place in line in advance with your phone and get a notification when it's your turn!"
+	m "Or you could, you know, spend some time with me while you wait ehehe~"
+	Menu:
+		m "Do you know about the stuff the people that run these places do to make lines more bearable?"
```

Comment:
Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.